### ISSUE TITLE
Color of password input in modal lacks contrast

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/manifold#2649]
![image](https://user-images.githubusercontent.com/255909/76256871-db34f800-620d-11ea-9096-5570bfd20b5d.png)
